src/brogue/SeedCatalog.c

```diff
@@ -76,15 +99,22 @@ static void printSeedCatalogItem(item *theItem, creature *theMonster, boolean is
 
     if (isCsvFormat) {
         printSeedCatalogCsvLine(rogue.seed, rogue.depthLevel, theItem->quantity, categoryName, kindName, enchantment,
-                                runicName, vaultNumber, opensVaultNumber, carriedByMonsterName, "");
+                                runicName, vaultNumber, opensVaultNumber, carriedByMonsterName, NO_ALLY_STATUS_STRING,
+                                mutationName);
     } else {
         upperCase(inGameItemName);
-        printf("        %s%s%s%s\n", inGameItemName, carriedByMonsterName, vaultNumber, opensVaultNumber);
+        if (theMonster != NULL) {
+            printf("        %s (%s)%s%s\n", inGameItemName, carriedByMonsterName, vaultNumber, opensVaultNumber);
+        } else {
+            printf("        %s%s%s\n", inGameItemName, vaultNumber, opensVaultNumber);
+        }
     }
 }
 
 static void printSeedCatalogMonster(creature *theMonster, boolean isCsvFormat) {
-    char categoryName[10] = "", allyStatusName[20] = "";
+    char categoryName[10] = "", allyStatusName[20] = "", mutationName[100] = "", theMonsterName[100] = "";
+
+    strcpy(mutationName, theMonster->mutationIndex >= 0 ? mutationCatalog[theMonster->mutationIndex].title : "");
```

Comment:
As an unrequested tip, depending on your 'taste' you might consider conditionally executing the strcpy as clearer than conditionally having the strcpy be a no-op.

```suggestion
    if (theMonster->mutationIndex >= 0) strcpy(mutationName, mutationCatalog[theMonster->mutationIndex].title);
```